ideascube/search/migrations/0002_reindex.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, connections
-from ideascube.search.utils import reindex_content
-
-
-def update_reindex(*args):
-    cursor_default = connections['default'].cursor()
-    cursor_default.execute("DROP TABLE IF EXISTS idx")
```

Comment:
Oh, I see what you mean. You're right, I completely missed that it was deleting the table in the **default** database.

So maybe instead of completely removing `0002_reindex`, we should transform it into `0002_remove_old_index_on_default_db`?